app/models/transaction.rb

```diff
@@ -19,6 +20,8 @@ class Transaction < ApplicationRecord
 
   before_validation :set_financial_quarter_from_date
 
+  scope :actual, -> { left_outer_joins(:adjustment).where("transaction_id IS NULL") }
```

Comment:
This way of retrieving "actual" transactions by defining them against another entity seems a bit brittle and prone to errors. 😬  Is this an intermediate stage before going to another paradigm?

It's making "actuals" second class citizens, in a way.